content/websockets/gateways.md

```diff
@@ -128,6 +128,8 @@ handleEvent(data) {
 
 > info **Hint** The `WsResponse` interface is imported from `@nestjs/websockets` package.
 
+> warning **Warning** You should return a class instance that `implements WsResponse` if your `data` field relies on Nest's serialization, as it ignores plain JavaScript objects responses.
```

Comment:
We should explicitly mention the `ClassSerializerInterceptor` here as "Nest's serialization" seems to be somewhat ambigious